### PR TITLE
Kaptain 2.0.0 Docs release-notes folder

### PR DESCRIPTION
**Pull request by juliangieseke**
Kaptain 2.0.0 docs "child PR" for #4309 (check its description)

### CODE REVIEW

File: pages/dkp/kaptain/2.0.0/release-notes/1.3.0/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+# Release Summary
+This release provides new features and capabilities that improve user experience, resolves reported issues, and maintains compatibility and support for upgraded packages. This release supports Kubeflow 1.4 and a new version of the Kaptain SDK, 1.0.0. For more information on SDK 1.0.0, see [SDK release notes](../../sdk/1.0.x/release-notes).
+
+## New features and capabilities
+
+### Garbage Collection
+
+Kaptain 1.3 supports automatic cleanup of completed and idling workloads. This prevents new workflows or versioned deployments from encountering a lack of resources and unnecessary pressure on controllers. Users will not reach namespace quotas, often set by administrators, as quickly. 
+Garbage collection allows users to configure cleanup of Kubeflow Pipeline runs and culling for Jupyter notebooks. Resources are automatically cleaned up when a cell execution is interrupted. Also, using the SDK, users can see how certain resources were created (e.g., by a user or by the SDK).
```
Comment (juliangieseke) [inline pages/dkp/kaptain/2.0.0/release-notes/1.3.0/index.md:22]:
```suggestion
Garbage collection allows users to configure cleanup of Kubeflow Pipeline runs and culling for Jupyter notebooks. Resources are automatically cleaned up when a cell execution is interrupted. Also, using the SDK, users can see how certain resources were created (for example by a user or by the SDK).
```

File: pages/dkp/kaptain/2.0.0/release-notes/1.3.0/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+For information on what resources are created during each phase of the model development lifecycle and how to use garbage collection, see [SDK](../../sdk/1.0.x/) and [SDK Troubleshooting](../../sdk/1.0.x/troubleshooting/).
+For examples of garbage collection for Katib experiments, see the [Hyperparameter Tutorial](../../tutorials/katib/).
+
+### Support for model building/deploying using non-deep learning frameworks
+
+With Kaptain SDK version 1.0.0, users can build and deploy models using frameworks even if they aren't supported in Kaptain. The SDK API is made more generic by using Kubernetes Jobs.
```
Comment (juliangieseke) [inline pages/dkp/kaptain/2.0.0/release-notes/1.3.0/index.md:28]:
```suggestion
With Kaptain SDK version 1.0.0, users can build and deploy models using frameworks even if they are not supported in Kaptain. The SDK API is made more generic by using Kubernetes Jobs.
```

File: pages/dkp/kaptain/2.0.0/release-notes/1.3.0/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+### Kaptain support on DKP 2.1
+
+Kaptain 1.3 is supported on version 2.1 of DKP, D2iQ's enterprise Kubernetes platform, in both non-air-gapped and air-gapped environments. For more information on DKP, view Konvoy and Kommander documentation on [docs.d2iq.com](https://docs.d2iq.com/).
+In DKP 2.1, Kaptain's dependencies are managed through Kommander, the centralized management plane of DKP which handles application deployments.
+
+## Fixes and Improvements
+
+* Fixed Kaptain redirectUrl is missing the hostname (COPS-7138)
+
+* Fixed the issue with a notebook doesn't mount notebook volume on the correct mountpoint (COPS-7064)
```
Comment (juliangieseke) [inline pages/dkp/kaptain/2.0.0/release-notes/1.3.0/index.md:44]:
```suggestion
* Fixed the issue with a notebook does not mount notebook volume on the correct mount point (COPS-7064)
```

**Comment by juliangieseke** [comment]
vale complaining about names..

File: pages/dkp/kaptain/2.0.0/release-notes/1.3.0/index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+With Kaptain SDK version 1.0.0, users can build and deploy models using frameworks even if they are not supported in Kaptain. The SDK API is made more generic by using Kubernetes Jobs.
+For an example using scikit-learn, see [Quickstart Tutorial](../../tutorials/sdk/quick-start/)
+
+### Models page
+
+Kaptain version 1.3 has a new 'Models' page for managing model servers.
+
+### Kaptain support on DKP 2.1
+
+Kaptain 1.3 is supported on version 2.1 of DKP, D2iQ's enterprise Kubernetes platform, in both non-air-gapped and air-gapped environments. For more information on DKP, view Konvoy and Kommander documentation on [docs.d2iq.com](https://docs.d2iq.com/).
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.0.0/release-notes/1.3.0/index.md:37]:
```suggestion
Kaptain 1.3 is supported on version 2.1 of DKP, D2iQ's enterprise Kubernetes platform, in both non air-gapped and air-gapped environments. For more information on DKP, view Konvoy and Kommander documentation on [docs.d2iq.com](https://docs.d2iq.com/).
```

**Pull request merged**